scripts/mkimage

```diff
@@ -327,6 +327,10 @@ fi # bootloader
         $PROJECT_DIR/$PROJECT/config/ovf.template > ${DISK/img/ovf}
     # combine ovf and vmdk into official ova
     tar -C $TARGET_IMG -cf ${DISK/img/ova} $(basename ${DISK/img/ovf}) $(basename ${DISK/img/vmdk})
+    # create sha256 checksum of ova image
+    ( cd $TARGET_IMG
+      sha256sum $(basename ${DISK/img/ova}) > $(basename ${DISK/img/ova}).sha256
+    )
```

Comment:
It's not new to this PR, but the `${DISK/img/ova}` substitution looks rather fragile - I know it's designed to replace the `.img` extension but if `img` appears anywhere in `$DISK` (which is based on `$TARGET_IMG/$IMAGE_NAME.img `, ie. an absolute path) then it will fail - for instance `/home/jimgarner/LibreELEC.tv/target/LibreELEC-RPi2.arm-9.0-devel-20180424011910-r12345-g6ce3be6.img ` will fail.

@Hiassoft can I suggest we make this a little more robust? I think adding DISK_TMP, DISK_OVA, DISK_OVF and DISK_VMDK variables where `DISK` is initialised in line #46 would allow all the DISK substitutions to be removed entirely, and be much safer/reliable. Or add a "bare" DISK_NOEXT variable without the `.img` extension, so the appropriate extension can be added where required. Given the number of substitutions (14 or so) this may be better done in a separate PR.

Reply:
agreed, this is terrible and needs changing. I'll look at it